android/services/mongodb-remote-perftests/src/androidTest/java/com/mongodb/stitch/android/services/mongodb/remote/SyncL2ROnlyPerformanceTests.kt

```diff
@@ -0,0 +1,322 @@
+package com.mongodb.stitch.android.services.mongodb.remote
+
+import android.util.Log
+import com.google.android.gms.tasks.Tasks
+import com.mongodb.stitch.core.services.mongodb.remote.sync.DefaultSyncConflictResolvers
+import org.bson.Document
+import org.bson.types.ObjectId
+import org.junit.Test
+import kotlin.random.Random
+
+import com.mongodb.stitch.core.services.mongodb.remote.ExceptionListener
+import org.junit.After
+import org.junit.Before
+
```

Comment:
Do we want tests to be in different files? In that case the runId's will be different for every test which is not what we wanted. ** Adam and I spoke offline about perhaps keeping the logic of this class and then having a devoted test file that runs the tests and calls the helper functions defined in here

Reply:
I'm happy to have a single class, and abstract the L2R stuff in a proxy file that we'll base the other scenario files on. I'll do that when addressing your comments.

Reply:
I do not think we should have a single class. There are simple ways to share a runId if need be. We don't need another monolith, not to mention the OOM issues we've been facing.

Reply:
by the time I saw this, it had already been done... but I did it in such a way that the definitions are in separate files, so I wouldn't quite call it a monolith. If we run into the OOM issue again, we can fix this up by having each test definition class handle its own test harness